Do not use SHALL and SHALL NOT (#4055)
open-telemetry/opentelemetry-specification: specification/versioning-and-stability.md

These terms do not refer to the specification or the Collector in this document.

Each language implementation MUST take these versioning and stability requirements, and produce a language-specific document which details how these requirements will be met.

> This document MUST be placed in the root of each repo and named `VERSIONING`.

Reviewer: ```suggestion
This document MUST be placed in the root of each repo and named `VERSIONING` or `VERSIONING.md`.
```

Signals MAY eventually be replaced. When this happens, they are marked as deprecated.

> Signals MUST only be marked as deprecated when the replacement becomes stable.

Reviewer: Maybe:

```suggestion
Signals MUST NOT be marked as deprecated unless the replacement is stable.
```
